content/hi/docs/reference/_index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: संदर्भ
+linkTitle: "संदर्भ"
+main_menu: true
+weight: 70
+content_type: concept
+no_list: true
+---
+
+
+<!-- overview -->
+
+कुबेरनेट्स प्रलेखन के इस खंड में संदर्भ हैं।
+
+
+
+<!-- body -->
+
+## API संदर्भ
+
+* [शब्दकोष](/docs/reference/glossary/) - कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची।
+* [कुबेरनेट्स API संदर्भ](/docs/reference/kubernetes-api/)
+* [कुबेरनेट्स के लिए एक-पृष्ठ API संदर्भ {{< param "version" >}}](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+* [कुबेरनेट्स API का उपयोग करना](/docs/reference/using-api/) - कुबेरनेट्स के लिए API का अवलोकन।
+* [API अभिगम नियंत्रण](/docs/reference/access-authn-authz/) - कुबेरनेट्स API एक्सेस को कैसे नियंत्रित करता है, इस पर विवरण।
+* [जाने-माने लेबल, टिप्पणी और दाग](/docs/reference/labels-annotations-taints/)
+
+## आधिकारिक तौर पर समर्थित ग्राहक पुस्तकालय
+
+कुबेरनेट्स API को प्रोग्रामिंग भाषा से कॉल करने के लिए, आप उपयोग कर सकते हैं
+[client libraries](/docs/reference/using-api/client-libraries/). आधिकारिक तौर पर समर्थित
+client libraries:
+
+- [कुबेरनेट्स Go client library](https://github.com/kubernetes/client-go/)
+- [कुबेरनेट्स Python client library](https://github.com/kubernetes-client/python)
+- [कुबेरनेट्स Java client library](https://github.com/kubernetes-client/java)
+- [कुबेरनेट्स JavaScript client library](https://github.com/kubernetes-client/javascript)
+- [कुबेरनेट्स C# client library](https://github.com/kubernetes-client/csharp)
+- [कुबेरनेट्स Haskell client library](https://github.com/kubernetes-client/haskell)
+
+## CLI
+
+* [kubectl](/docs/reference/kubectl/overview/) - कमांड चलाने और कुबेरनेट्स क्लस्टर के प्रबंधन के लिए मुख्य CLI उपकरण।
+    * [JSONPath](/docs/reference/kubectl/jsonpath/) - kubectl के साथ प्रयोग करने के लिए 
+वाक्य - विन्यास मार्गदर्शक [JSONPath expressions](https://goessner.net/articles/JsonPath/).
+* [kubeadm](/docs/reference/setup-tools/kubeadm/) - एक सुरक्षित कुबेरनेट्स क्लस्टर का आसानी से प्रावधान करने के लिए CLI टूल।
+
+## अवयव
+
+* [kubelet](/docs/reference/command-line-tools-reference/kubelet/) - प्राथमिक एजेंट जो प्रत्येक   नोड पर चलता है। क्यूबलेट पॉडस्पेक्स का एक सेट लेता है
+  और यह सुनिश्चित करता है कि वर्णित कंटेनर चल रहे हैं और स्वस्थ हैं।
+* [kube-apiserver](/docs/reference/command-line-tools-reference/kube-apiserver/) -
+  REST API जो API वस्तु जैसे पॉड्स, सेवाएं, प्रतिकृति नियंत्रक के लिए डेटा को मान्य और कॉन्फ़िगर करता है।
+* [kube-controller-manager](/docs/reference/command-line-tools-reference/kube-controller-manager/) - डेमॉन जो कुबेरनेट्स के साथ भेजे गए कोर कंट्रोल लूप्स को एम्बेड करता है।
+* [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) - बैक-एंड के एक सेट में साधारण TCP/UDP स्ट्रीम फॉरवर्डिंग या राउंड-रॉबिन TCP/UDP फॉरवर्डिंग कर सकते हैं। 
```

Comment:
The word `forwarding` can be replaced with `अग्रेषण` or atleast we provide both the words with `या`.

Reply:
"forwarding" is kind of a technical term in networking (for eg: port forwarding etc.), So I believe translating it literally would create confusions. Better to have transliteration instead. Thoughts?

Reply:
Yes I thought about the same and that's why I mentioned about providing both the terms by 'or' because for many Hindi speakers, forwarding would be difficult to understand although it's the right term.

Reply:
I have added even simpler `आगे भेजना` in parenthesis next to फॉरवर्डिंग in latest commits. अग्रेषण is not very descriptive and could lead to confusion with english word aggression.